pkg/pb/logservice/logservice.pb.go

```diff
@@ -385,12 +385,12 @@ func (ServiceType) EnumDescriptor() ([]byte, []int) {
 }
 
 type CNStore struct {
-	UUID                 string          `protobuf:"bytes,1,opt,name=UUID,proto3" json:"UUID,omitempty"`
```

Comment:
do not push pb.go file which the proto file is not changed.

Reply:
The previous version was 3.21.7, Not the recommended version, I will submit it along with the document location changes after I regenerate it.